imagetest/test_suites/metadata/shutdown_script_test.go

```diff
@@ -0,0 +1,60 @@
+
+package metadata
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const shutdownTime = 110 // about 2 minutes
+
+// TestGuestShutdownScript test that shutdown scripts can run for around two minutes
+func TestGuestShutdownScript(t *testing.T) {
+	// second boot
+	bytes, err := ioutil.ReadFile("/shutdown.txt")
+	if err != nil {
+		t.Fatalf("error reading file: %v", err)
+	}
+	lines := strings.Split(strings.TrimSpace(string(bytes)), "\n")
+	if len(lines) < shutdownTime {
+		t.Fatalf("shut down time less than %d seconds.", shutdownTime)
+	}
+}
+
+// TestShutdownScript test the standard metadata script.
+func TestShutdownScript(t *testing.T) {
+	// second boot
+	bytes, err := ioutil.ReadFile(shutdownOutputPath)
+	if err != nil {
+		t.Fatalf("failed to read shutdown output %v", err)
+	}
+	output := strings.TrimSpace(string(bytes))
+	if output != shutdownContent {
+		t.Fatalf(`shutdown script output expect "%s", but actually "%s"`, shutdownContent, output)
+	}
+}
+
+// TestRandomShutdownScriptNotCrashVM test that a script with random content
+// doesn't crash the vm.
+func TestRandomShutdownScriptNotCrashVM(t *testing.T) {
+	// second boot
```

Comment:
confusing comment. this code will be run on both boots

Reply:
the comment is just incorrect. this is not always the second boot. the comment also doesn't convey any meaningful information

Reply:
I remove the comment